### PR TITLE
refactor: fix translations in localization files

### PR DESCRIPTION
**Pull request by suschneider**
## PR Type
[x] Other: translations

## What Is the Current Behavior?
The technical writers revised our translations.

## What Is the New Behavior?
Translations are fixed.

## Does this PR Introduce a Breaking Change?
[x] No

[AB#71449](https://dev.azure.com/intershop-com/cefd1005-00a7-4c79-927f-a16947d1b2e6/_workitems/edit/71449)

### CODE REVIEW

File: src/assets/i18n/fr_FR.json [hunk 1]
```diff
@@ -459,19 +459,19 @@
...
+  "approval.details.breadcrumb.order.label": "Demande d’achat ",
   "approval.details.conditions": "Votre demande d’achat doit être approuvée avant qu’elle soit envoyée, puisqu’au moins une de ces conditions a été remplie :",
   "approval.details.conditions.budget_limit": "La demande d’achat dépasse votre budget.",
   "approval.details.conditions.cost_center": "La demande d’achat est attribuée à un centre de coûts.",
-  "approval.details.conditions.order_spend_limit": "La demande d’achat dépasse votre limite de dépenses de la commande.",
-  "approval.details.contacts.heading": "Contacts de l’approbateur",
-  "approval.details.cost_center_approvers.people_allowed": "La personne suivante est autorisée à approuver votre demande d’achat, car elle est associée au centre de coûts {{0}}:",
+  "approval.details.conditions.order_spend_limit": "La demande d’achat dépasse votre limite de dépenses par commande.",
+  "approval.details.contacts.heading": "Contacts des approbateurs",
+  "approval.details.cost_center_approvers.people_allowed": "Les personnes suivantes sont autorisées à approuver votre demande d’achat, lorsque la limite de dépenses ou le budget sont dépassés :",
```
Comment (SGrueber) [inline src/assets/i18n/fr_FR.json:471]: parameter for the cost center name is missing now

File: src/assets/i18n/fr_FR.json [hunk 2]
```diff
@@ -756,7 +756,7 @@
   "modal.heading.remove.order.template": "Supprimer le produit du modèle de commande",
   "order.tracking.error": "Malheureusement, nous n’avons pas pu localiser de commande avec les informations que vous nous avez fournies.",
   "order_template.create.heading": "Créer un modèle de commande",
-  "payment.error.PaymentInstrumentAlreadyExists": "L’instrument de paiement n’a pas pu être créé. Les données de paiement avec les paramètres fournis existent déjà.",
+  "payment.error.PaymentInstrumentAlreadyExists": "Le moyen de paiement n’a pas pu être mis en place. Les données de paiement avec les paramètres fournis existent déjà.",
```
Comment (SGrueber) [inline src/assets/i18n/fr_FR.json:759]: If you change the translation for (payment instrument - Zahlungsmittel) here - it should be done for all other occurrances, too. Pls note the difference between payment method and payment instrument.

**Review by SGrueber — CHANGES_REQUESTED** [review]

File: src/assets/i18n/fr_FR.json (hunk 2 above)
Comment (mglatter) [reply src/assets/i18n/fr_FR.json:759]: You are right. We will change "l'instrument de paiement" to "le moyen de paiement" at all instances occurring in the localization file within beginning of this week. Please wait untilthis is done before merging.

**Review by mglatter — COMMENTED** [review]
We will change "l'instrument de paiement" to "le moyen de paiement" at all instances occurring in the localization file within beginning of this week. Please wait until this is done before merging.

File: src/assets/i18n/fr_FR.json (hunk 2 above)
Comment (mglatter) [reply src/assets/i18n/fr_FR.json:759]: Done

**Review by mglatter — APPROVED** [review]

**Review by SGrueber — APPROVED** [review]

**Pull request merged**